src/replica/replica_config.cpp

```diff
@@ -594,6 +607,31 @@ void replica::update_ac_allowed_users(const std::map<std::string, std::string> &
     _access_controller->update(allowed_users);
 }
 
+void replica::update_allow_ingest_behind(const std::map<std::string, std::string> &envs)
+{
+    bool new_value = false;
+    if (!get_bool_envs(envs, replica_envs::ROCKSDB_ALLOW_INGEST_BEHIND, new_value)) {
+        return;
+    }
+    if (new_value != _allow_ingest_behind) {
+        // update it into file
+        auto info = _app_info;
```

Comment:
implement a function named like update_envs_in_info_file, and put L618->L622 into this func

Reply:
There are other code use similar code, I will add it in another pr, I add comment here.